src/openforms/js/components/admin/form_design/logic/actions/dmn/VariableMapping.js

```diff
@@ -65,8 +61,11 @@ const VariableMapping = ({mappingName, values, formVariables}) => {
                       htmlFor={`${mappingName}.${index}.dmnVariable`}
                       name={`${mappingName}.${index}.dmnVariable`}
                     >
-                      <TextInput
+                      <Select
                         id={`${mappingName}.${index}.dmnVariable`}
+                        allowBlank={true}
```

Comment:
```suggestion
                        allowBlank
```
JSX makes this `true` :)